src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const carRent = 40;
+
+  if (days < 3) {
+    return carRent * days;
+  } else if (days >= 3 && days < 7) {
+    return (carRent * days) - 20;
```

Comment:
It's better to create variables for every value (such as 20) to improve readability and maintenance. 
E.g `const threeDayDiscount = 20;`